src/components/DescriptionList/DescriptionList.scss

```diff
@@ -41,6 +53,10 @@ $breakpoint: 550px;
     flex: 1 1 51%;
     padding: spacing() 0;
 
+    .spacingTight & {
+      padding: spacing(tight) 0;
+    }
```

Comment:
Do you still need to add styles for loose spacing?

Reply:
Oh it looks like loose is the default. That seems odd to me. Do we need a middle ground or do you think this is ok for now?

Reply:
Loose is the default. Didn't want to make it a boolean prop so add loose. Not needed but allows us to add more in the future without breaking. thoughts?

Reply:
open to suggestions though